rom/laghos_rom.cpp

```diff
@@ -608,8 +971,17 @@ void ROM_Sampler::SetupEQP_Force(const CAROM::Matrix* snapX, const CAROM::Matrix
     MFEM_VERIFY(snapV->numRows() == input.H1FESpace->GetTrueVSize(), "");
     MFEM_VERIFY(snapE->numRows() == input.L2FESpace->GetTrueVSize(), "");
 
-    SetupEQP_Force_Eq(snapX, snapV, snapE, basisV, basisE, input, false);
-    SetupEQP_Force_Eq(snapX, snapV, snapE, basisV, basisE, input, true);
+	if (input.hyperreductionSamplingType == eqp)
+	{
+		// basic EQP: different rules for velocity and energy
+		SetupEQP_Force_Eq(snapX, snapV, snapE, basisV, basisE, input, false);
+		SetupEQP_Force_Eq(snapX, snapV, snapE, basisV, basisE, input, true);
+	}
+	else if (input.hyperreductionSamplingType == eqp_energy)
```

Comment:
Where should I add the hyperreduction option `eqp_energy` so that it's a valid option? This is the only part of the code that gives me a compilation error.

Reply:
I believe I resolved that in commit c4280af by adding the new option in `laghos_rom.hpp`.